platforms/documentation/docs/src/docs/userguide/authoring-builds/tasks/more_about_tasks.adoc

```diff
@@ -484,177 +476,68 @@ include::{snippetsPath}/tasks/finalizers/tests/taskFinalizers.out[]
+To specify a finalizer task you use the link:{groovyDslPath}/org.gradle.api.Task.html#org.gradle.api.Task:finalizedBy(java.lang.Object++[]++)[Task.finalizedBy(java.lang.Object...)] method.
+This method accepts a task instance, a task name, or any other input accepted by link:{groovyDslPath}/org.gradle.api.Task.html#org.gradle.api.Task:dependsOn(java.lang.Object++[]++)[Task.dependsOn(java.lang.Object...)].
 
 [[sec:lifecycle_tasks]]
 == Lifecycle tasks
 
-Lifecycle tasks are tasks that do not do work themselves. They typically do not have any task actions. Lifecycle tasks can represent several concepts:
+Lifecycle tasks are tasks that do not do work themselves.
+They typically do not have any task actions.
+
+Lifecycle tasks can represent several concepts:
 
 * a work-flow step (e.g., run all checks with `check`)
 * a buildable thing (e.g., create a debug 32-bit executable for native components with `debug32MainExecutable`)
 * a convenience task to execute many of the same logical tasks (e.g., run all compilation tasks with `compileAll`)
 
-The Base Plugin defines several <<base_plugin#sec:base_tasks,standard lifecycle tasks>>, such as `build`, `assemble`, and `check`. All the core language plugins, like the <<java_plugin#java_plugin,Java Plugin>>, apply the Base Plugin and hence have the same base set of lifecycle tasks.
+The Base Plugin defines several <<base_plugin#sec:base_tasks,standard lifecycle tasks>>, such as `build`, `assemble`, and `check`.
+All the core language plugins, like the <<java_plugin#java_plugin,Java Plugin>>, apply the Base Plugin and hence have the same base set of lifecycle tasks.
 
-Unless a lifecycle task has actions, its <<#sec:task_outcomes,outcome>> is determined by its task dependencies. If any of those dependencies are executed, the lifecycle task will be considered `EXECUTED`. If all of the task dependencies are up to date, skipped or from cache, the lifecycle task will be considered `UP-TO-DATE`.
+Unless a lifecycle task has actions, its <<#sec:task_outcomes,outcome>> is determined by its task dependencies.
+If any of those dependencies are executed, the lifecycle task will be considered `EXECUTED`.
+If all the task dependencies are `UP-TO-DATE`, `SKIPPED` or `FROM-CACHE` the lifecycle task will be considered `UP-TO-DATE`.
 
 [[sec:the_idea_behind_gradle_tasks]]
-== Summary
-
-If you are coming from Ant, an enhanced Gradle task like _Copy_ seems like a cross between an Ant target and an Ant task. Although Ant's tasks and targets are really different entities, Gradle combines these notions into a single entity. Simple Gradle tasks are like Ant's targets, but enhanced Gradle tasks also include aspects of Ant tasks. All of Gradle's tasks share a common API and you can create dependencies between them. These tasks are much easier to configure than an Ant task. They make full use of the type system, and are more expressive and easier to maintain.
-
-=== Moved documentation
```

Comment:
These are not used at all in the current documentation. These links are only there in case of external links and land in the new task section, which is where we want our users to be.